### ISSUE TITLE
[Bug] Lockwise does not prevent access to websites credentials #16288 REOPENED

### ISSUE THREAD

**Issue by NJTH317** [mozilla-mobile/fenix#16355]
Initial problem: after installing Lockwise and going to any site I've stored username/password, I'm invited to open Lockwise AND, below, get 'suggested logins" with my username and password displayed. Obviously, this is a problem - Firefox should prevent this from displaying until I sign in to Lockwise.

I tried uninstalling/reinstalling Firefox - now using Firefox for Android 82.1.1 (Build #2015770923). Same problem.

After advice from ekager in previous thread, I tried turning off autofill. this did not work - passwords etc. still offered under "suggested logins" even though I have not signed in to Lockwise, as screenshot shows.

![Screenshot](https://user-images.githubusercontent.com/73712279/98040391-0ee41400-1dd5-11eb-9b29-f335e9abb04d.jpg)

Bottom line - Firefox is not protecting my usernames/passwords, even with Lockwise.

For some reason, previous thread closed without a solution - I would appreciate a solution.

Thanks!

### Device information

* Android device: ?
* Fenix version: ?

**Comment by ekager** [comment]
Please do not open new issues like this closing for #16288
